### PR TITLE
Updating Swift tests to remove outdated tests and use more current calls

### PR DESCRIPTION
**Pull request by drewmaxwell-apple**
Changes now use CoreImage rather than Foundation, or make calls to implicitly import Foundation.

### CODE REVIEW

**Review by shahmishal — APPROVED** [review]

**Comment by shahmishal** [comment]
@swift-ci test

**Comment by shahmishal** [comment]
```
Failed Tests (1):
  Swift(macosx-x86_64) :: Interpreter/SDK/Foundation_NSRect.swift
```

File: test/Interpreter/SDK/Foundation_NSRect.swift [hunk 1]
```diff
@@ -4,7 +4,9 @@
 // iOS doesn't have NSRect. iOS and OS X CGRect is tested elsewhere.
 // REQUIRES: OS=macosx
 
-import Foundation
+import Coreimage
```
Comment (shahmishal) [inline test/Interpreter/SDK/Foundation_NSRect.swift:7]:
```suggestion
import CoreImage
```

**Comment by shahmishal** [comment]
@swift-ci test

**Review by mikeash — APPROVED** [review]

**Pull request merged**